compiler/src/iree/compiler/Codegen/Dialect/PCF/IR/PCFOps.td

```diff
@@ -0,0 +1,894 @@
+// Copyright 2025 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef IREE_CODEGEN_DIALECT_PCF_OPS
+#define IREE_CODEGEN_DIALECT_PCF_OPS
+
+include "iree/compiler/Codegen/Dialect/PCF/IR/PCFBase.td"
+include "mlir/IR/BuiltinAttributeInterfaces.td"
+include "mlir/IR/OpAsmInterface.td"
+include "mlir/IR/Properties.td"
+include "mlir/IR/SymbolInterfaces.td"
+include "mlir/Interfaces/ControlFlowInterfaces.td"
+include "mlir/Interfaces/InferTypeOpInterface.td"
+include "mlir/Interfaces/SideEffectInterfaces.td"
+include "mlir/Interfaces/ViewLikeInterface.td"
+
+//===----------------------------------------------------------------------===//
+// Alloc
+//===----------------------------------------------------------------------===//
+
+def OpGroupAllocOps : OpDocGroup {
+  let summary = "Alloc ops";
+  let description = "";
+}
+
+let opDocGroup = OpGroupAllocOps in {
+
+//===----------------------------------------------------------------------===//
+// AllocOp
+//===----------------------------------------------------------------------===//
+
+def AllocOp : PCF_Op<"alloc", []> {
+  let summary = [{
+    Shaped ref allocation operation
+  }];
+  let description = [{
+    Allocates a `pcf.sref` with the given element type and shape. Dynamic
```

Comment:
Is it `pcf.sref` or `pcf.shaped_ref`?

Reply:
Removed all instances of `pcf.shaped_ref`, that was old and I missed it. I went with shortening the printed type name to keep the IR succinct, but I can see the argument for just using the long form name everywhere (tbc if you wanted the long form name that would be a bit of a pain updating all of the tests but whatever). My bot didn't struggle at all with the naming incongruity so I kept it.